Tiny.RestClient/TinyRestClient.cs

```diff
@@ -288,6 +301,8 @@ public IRequest DeleteRequest(string route = null)
            Request tinyRequest,
            CancellationToken cancellationToken)
         {
+            await ExecuteDynamicHeadersAsync(tinyRequest);
```

Comment:
ExecuteDynamicHeadersAsync must be executed on SendRequestAsync function just before to add headers